Automattic/wp-calypso
# Remove Jetpack Starter from pricing page

## Proposed Changes

* Remove Jetpack Starter pricing card from the pricing page
* Remove some backend logic related to Jetpack Starter
* Remove Starter from the comparison page

P2: pbNhbs-7hz-p2

## Testing Instructions

1. Use the Calypso Green live branch
2. Go to `/pricing`
3. Make sure Jetpack Starter does not show up
![image](https://github.com/Automattic/wp-calypso/assets/65001528/1c3a1c74-fd31-417b-ad55-53b464bce512)
4. Go to `/features/comparison` and make sure Starter does not show up here as well
![image](https://github.com/Automattic/wp-calypso/assets/65001528/7ed9e540-61f7-4d40-b9e2-f1c61c951568)
5. Make sure the title and subtitle have been updated to only include bundles
![image](https://github.com/Automattic/wp-calypso/assets/65001528/ed6f3c03-69a8-4bdb-bcdd-a35c8e19f56c)
6. If you do not already have a Jetpack Starter subscription, go to https://wordpress.com/checkout/jetpack/jetpack_starter_yearly and purchase one. Then go to `/me/purchases` and make sure the plan loads correctly here
![image](https://github.com/Automattic/wp-calypso/assets/65001528/117cb8c5-47c4-4d4a-9814-6033b39ec4ff)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/618e1f2c-19c2-4e71-80e6-8d6b7af3ffd6)

## Pre-merge Checklist
